### PR TITLE
[aes:testplan] Updated test plan with action items from V1 review

### PR DESCRIPTION
**Pull request by rasmus-madsen**
I think this the last PR before #1401 can be merged

### CODE REVIEW

File: hw/ip/aes/data/aes_testplan.hjson [hunk 1]
```diff
@@ -75,6 +75,21 @@
             - Write unsupported configurations (Key length and mode are 1 hot, what happens if more than one bit is set.)'''
       milestone: V2
       tests: []
+    }
+	{
```
Comment (sriyerg) [inline hw/ip/aes/data/aes_testplan.hjson]: nit: brackets and desc are misaligned. Otherwise, LGTM.

File: hw/ip/aes/data/aes_testplan.hjson [hunk 2]
```diff
@@ -75,6 +75,21 @@
...
+      name: trigger_clear_test
+      desc: '''
+            Exercise trigger and clear registers at random times to make sure we handle the different cornercases correctly.
+			Example of a cornercases clearing data input or data output before the data is consumed or the DUT finishes an operation.'''
```
Comment (vogelpi) [inline hw/ip/aes/data/aes_testplan.hjson]: I think a tab has sneaked in here.

File: hw/ip/aes/data/aes_testplan.hjson [hunk 3]
```diff
@@ -75,6 +75,21 @@
...
+      milestone: V2
+      tests: []
+    }
+	{
```
Comment (vogelpi) [inline hw/ip/aes/data/aes_testplan.hjson]: Another misaligned brackets.

File: hw/ip/aes/data/aes_testplan.hjson (hunk 1 above)
Comment (rasmus-madsen) [reply hw/ip/aes/data/aes_testplan.hjson]: solved - thank you

File: hw/ip/aes/data/aes_testplan.hjson (hunk 2 above)
Comment (rasmus-madsen) [reply hw/ip/aes/data/aes_testplan.hjson]: solved - thank you

File: hw/ip/aes/data/aes_testplan.hjson (hunk 3 above)
Comment (rasmus-madsen) [reply hw/ip/aes/data/aes_testplan.hjson]: solved - thank you

**Comment by rasmus-madsen** [comment]
@vogelpi @sriyerg sorry for the delay - my RSA key was somehow removed from my server so I could not pull or push to the remote. thank you pirmin for the support.

**Pull request merged**